### ISSUE TITLE
ml 0.6

### ISSUE THREAD

**Issue by nsulikowski** [dotnet/machinelearning-samples#61]
please convert these samples to the new 0.6 api

**Comment by CESARDELATORRE** [comment]
Sure, we're in the process of doing that. Please, check out the temporal branch that will be holding the migrated samples:
 https://github.com/dotnet/machinelearning-samples/tree/features/samples-new-api

For now, just the Getting started sample is migrated here:
https://github.com/dotnet/machinelearning-samples/tree/features/samples-new-api/samples/csharp/getting-started/BinaryClassification_SentimentAnalysis

More samples will be coming soon.

**Comment by mariuszwojcik** [comment]
Hi Cesar,

I plan to make myself more familiar with ML.NET and I will be willing to help with migration of those samples. Are you working on any other sample right now (just trying to avoid conflicts).

**Comment by CESARDELATORRE** [comment]
@mariuszwojcik - 👍 Thanks for your collaboration! Please, wait a few days more, ping me next week, ok?. We're working on a sample which can be the reference/model for migrating the rest. Also, still to be decided if we're going to migrate the samples to the "dynamic API" or the "static API". 
If you want to contact me directly, send me an email to: cesardl at microsoft.com

**Comment by mariuszwojcik** [comment]
@CESARDELATORRE - No problem. Will ping you back next week for progress update.

**Comment by CESARDELATORRE** [comment]
Closing this issue. We have migrated quite a few samples to v0.6 and dynamic API already. Note that there are still a few missing samples such as the F# samples and a few other we'll be adding.
See the new sample list/table:
https://github.com/dotnet/machinelearning-samples#mlnet-samples

I'm closing this issue but feel free to open issues related to specific samples to be migrated.
Contributions will be welcome! 👍
